src/custom-element.js

```diff
@@ -1570,8 +1569,7 @@ function createBaseCustomElementClass(win) {
         this.loadingContainer_.classList.toggle('amp-hidden', !state);
         this.loadingElement_.classList.toggle('amp-active', state);
 
-        if (!state && cleanup &&
-            !this.implementation_.isLoadingReused()) {
+        if (!state && !this.implementation_.isLoadingReused()) {
```

Comment:
I feel like not having `cleanup` will break https://github.com/ampproject/amphtml/blob/649b614fd399ca5e15e8460b94dffd1989b5e815/src/custom-element.js#L1106 where code is trying to just hide but not destroy the loading icon for all elements regardless of isLoadingReused. Now I am not sure when that particular line is called, it seems a bit strange to me since loading icon should be invisible by default anyway so I don't see how something can have layoutCount = 0 and need loading icons to be toggle off.

Reply:
Yup, a bunch of test expectations needed to be changed because of this too. I am happy to split out the `cleanup` removal in a separate PR since this is getting a bit convoluted?